modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/UserManager.java

```diff
@@ -46,8 +46,8 @@ public User getUser(String id, Map<String, Boolean> requiredAttributes)
     public void deleteUser(String userId)
             throws NotFoundException, CharonException, NotImplementedException, BadRequestException;
 
-    public List<Object> listUsersWithGET(Node node, int startIndex, int count,
-                                         String sortBy, String sortOrder, Map<String, Boolean> requiredAttributes)
+    public List<Object> listUsersWithGET(Node node, int startIndex, int count, String sortBy, String sortOrder,
+                                         String domainName, Map<String, Boolean> requiredAttributes)
```

Comment:
This is an API change. Have we considered how to handle existing deployments which are using the older method api?

Reply:
since we released this API changes with a newer vresion we don't need to maintain backward compatibility for those API?
ref https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/112#discussion_r320653559